### DISCUSSION TITLE
Add Listing Properties to "Includes" for get listing endpoints

### DISCUSSION THREAD

**Discussion by emergingdzns** [etsy/open-api#801] [Ideas]
This is a re-open as a discussion for https://github.com/etsy/open-api/issues/573.

It would be enormously helpful if the listing properties/attributes were available as an Include option like Inventory, Images, etc. @chrisrogala made a compelling business case. Ours is similar and would be helpful to complete migration to V3.

**Comment by simonj** [comment]
+1 for this

**Comment by mt-bt** [comment]
+1 for this
